Lib/configparser.py

```diff
@@ -1048,6 +1054,39 @@ def __iter__(self):
         # XXX does it break when underlying container state changed?
         return itertools.chain((self.default_section,), self._sections.keys())
 
+    def __str__(self):
+        config_dict = {
+            section: {key: value for key, value in self.items(section, raw=True)}
```

Comment:
```suggestion
            section: dict(self.items(section, raw=True))
```

It should be enough right?